jerry-core/debugger/debugger.c

```diff
@@ -227,8 +227,10 @@ jerry_debugger_sleep (unsigned milliseconds) /**< suspending time */
     milliseconds / 1000, (milliseconds % 1000) * 1000000L /* Seconds, nanoseconds */
   }
   , NULL);
-#elif defined (HAVE_UNISTD_H)
+#elif HAVE_UNISTD_H
```

Comment:
I don't agree either with the proposed change or with the review.
- `#elif HAVE_UNISTD_H` is not correct, as the CMakeLists.txt defines HAVE_UNISTD_H without a value (if it gets defined at all), so this conditional can never be evaluated as 1 (true).
- `#if defined (HAVE_UNISTD_H) && HAVE_UNISTD_H` is unidiomatic and incorrect (because of the same reason, i.e., the `&& HAVE_UNISTD_H` part will always be false.

IMHO, the original form (`#elif defined (HAVE_UNISTD_H)`) is the only correct approach